### PR TITLE
fix: add active styling to edit link bar link icon

### PR DESCRIPTION
**Pull request by arpitv96171**
Link bar link button container has active styling but the icon itself lacked the active styling. 
This PR adds the active styling to the link button in `EditLinkBar.tsx`

### CODE REVIEW

**Comment by arpitv96171** [comment]
updated patch version based on suggestion: https://github.com/10play/10tap-editor/pull/170#issuecomment-2254365308

Please let me know if it is to be reverted

If this patch version update is to be kept, this PR is to be merged before #173

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@10play/tentap-editor",
-  "version": "0.5.13",
+  "version": "0.5.14",
```
Comment (17Amir17) [inline package.json:3]: Hey @arpitv96171 no need to update the version - after you remove this, we're good to merge just make sure the commit has a valid name

Comment (arpitv96171) [reply package.json]: updated

**Review by 17Amir17 — APPROVED** [review]

**Comment by 17Amir17** [comment]
Fixes https://github.com/10play/10tap-editor/issues/169

**Pull request merged**